packages/std/src/imports.rs

```diff
@@ -12,15 +12,18 @@ use crate::traits::{Api, Querier, QuerierResponse, ReadonlyStorage, Storage};
 use crate::traits::{Order, KV};
 use crate::types::{CanonicalAddr, HumanAddr};
 
-// this is the buffer we pre-allocate in get - we should configure this somehow later
-static MAX_READ: usize = 2000;
+/// A kibi (kilo binary)
+static KI: usize = 1024;
+
+/// The number of bytes of the memory region we pre-allocate for the result data in Storage.get
+static RESULT_BUFFER_LENGTH: usize = 128 * KI;
+
+/// The number of bytes of the memory region we pre-allocate for the result data in queries
```

Comment:
Why not a bit bigger? If we are gong 128KB there, maybe 16KB here at least?

Actually listing all validators (planned external query) may be quite large. Say 150+ bytes per validator at 125 validators on cosmos hub may even exceed 16KB.

So let's throw 64KB here. (or 128KB).
This memory is deallocated after use, so once we make one query that grabs 2 new pages, those same pages are reused by all other calls.

Reply:
Increased to 128 KiB. It was just 4 before because it used to be 4000.

Reply:
Thanks. And I agree with the two different sizes, we can adjust those all later